Composer/packages/client/src/pages/botProject/RootBotExternalService.tsx

```diff
@@ -479,7 +478,6 @@ export const RootBotExternalService: React.FC<RootBotExternalServiceProps> = (pr
           />
         </div>
         <PrimaryButton
-          disabled={displayManageLuis || displayManageQNA}
```

Comment:
This seems a little risky to me. Is the problem that by the time the modal is dismissed, the buttons are still disabled so cannot get focused?

Reply:
Played with these buttons a bit: either currently focused button can't be remembered because it is disabled when modal gets opened (hence it cant be restored), or it can't be restored because the button is still disabled while modal is closing.
Tried different scenarios, and haven't found any regression with these buttons.